### PR TITLE
[4.0] Remove checkin, checkout methods from ProfileModel

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue # .

### Summary of Changes
Table  #__users does not have **checked_out** or **checked_out_time**, so the checkin, checkout method in ProfileModel are useless and could be removed

### Testing Instructions
1. Create a menu item to link to Edit User Profile menu option
2. Apply patch
3. Login to any account, access to that menu item to edit profile data
4. Press Save button, make sure the profile data is still being saved properly

### CODE REVIEW

File: components/com_users/src/Model/ProfileModel.php [hunk 1]
```diff
@@ -60,71 +60,6 @@ public function __construct($config = array(), MVCFactoryInterface $factory = nu
...
-	 */
-	public function checkin($userId = null)
-	{
-		// Get the user id.
-		$userId = (!empty($userId)) ? $userId : (int) $this->getState('user.id');
-
-		if ($userId)
-		{
-			// Initialise the table with Joomla\CMS\User\User.
-			$table = Table::getInstance('User', 'Joomla\\CMS\Table\\');
```
Comment (Quy) [inline components/com_users/src/Model/ProfileModel.php:80]: Please remove `use Joomla\CMS\Table\Table;`. Thanks.

Comment (joomdonation) [reply components/com_users/src/Model/ProfileModel.php:80]: @Quy Done.

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on e4acb46eeb45819921e5de2a58514d1d5ab4943f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32590">issues.joomla.org/tracker/joomla-cms/32590</a>.</sub>

**Comment by chmst** [comment]
Good find - but other question: Why has the user no checkin/checkout as all other items?

**Comment by joomdonation** [comment]
> Good find - but other question: Why has the user no checkin/checkout as all other items?

I'm unsure. I can only guess that it's because user is different from other content types in Joomla like articles, contacts....Usually, we do not have multiple users edit the same account at a time, so checkin, checkout is not required.

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on e4acb46eeb45819921e5de2a58514d1d5ab4943f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32590">issues.joomla.org/tracker/joomla-cms/32590</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32590">issues.joomla.org/tracker/joomla-cms/32590</a>.</sub>

**Pull request merged**